rules/windows/persistence_remote_password_reset.toml

```diff
@@ -18,7 +18,9 @@ index = ["winlogbeat-*", "logs-system.security*", "logs-windows.forwarded*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Account Password Reset Remotely"
-note = "This rule may cause medium to high performance impact due to logic scoping all remote Windows logon activity."
+note = """
+This rule may cause medium to high performance impact due to scoping all remote Windows logon activity within the first sequence. This rule by default will be noisy if not scoped to custom privileged accounts. If noisy, duplicate the rule and replace the `TargetUserName` or `TargetSid` filters with your own naming convention and accounts of interest.
```

Comment:
maybe add the `## Performance` header ? also adding a performance to the tags array will help filter rules via GUI.

Reply:
Added performance header. I'd like to discuss with the team more about tagging options. Do not want to block release.